src/components/combination-result/CombinationResult.jsx

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Awesome work my friend. In the case of the possibilities images you can use the `SVG` only instead of exporting all the card as a PNG file. Then, in the assets folder you can define an `svg` as a `jsx` component inside a svgs folder. For example:

```
import React from 'react';

export const CautionSVG = () => {
 return (
   <svg>...</svg>
 );
}
```

This way we keep smaller files and we have more control on styles. Also  svg's have a better definition (infinite) than a pixel image. One more thing is to be consistent on the naming of the files.

With the svg's components then you can create the CombinationResult component as a card.

Note: In figma you can directly export it as an svg.